### PR TITLE
Added Via-enabled keymap for Physix

### PR DESCRIPTION
**Pull request by Frothy**
## Description
Added a Via-enabled keymap for LZ Physix

Via PR: https://github.com/the-via/keyboards/pull/229

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] ~My change requires a change to the documentation.~
- [ ] ~I have updated the documentation accordingly.~
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] ~I have added tests to cover my changes.~
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Frothy** [comment]
@hineybush  -- hope you don't mind that I started this. Feel free to suggest changes or let me know if you already have something in-the-works and I can close this.

File: keyboards/hineybush/physix/keymaps/via/keymap.c [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+
```
Comment (fauxpark) [inline keyboards/hineybush/physix/keymaps/via/keymap.c:19]:
```suggestion
```

**Comment by noroadsleft** [comment]
Thanks!

**Comment by hineybush** [comment]
@Frothy I missed this earlier and just tried to merge the same thing. Thank you!

**Pull request merged**